<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeAssignment</class>
 <widget class="QDialog" name="AttributeAssignment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttributeAssignment</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_field">
     <property name="text">
      <string>Field</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_value">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_layer">
     <property name="text">
      <string>Layer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QgsFieldComboBox" name="mFieldComboBox"/>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="mValuePlaceholder" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui.QgsMapLayerComboBox</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui.QgsFieldComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
